tests/lib/reset.sh

```diff
@@ -145,6 +145,13 @@ reset_all_snap() {
     if [ "$1" != "--keep-stopped" ]; then
         systemctl start snapd.service snapd.socket
     fi
+
+    # Exit in case there is a snap in broken state after restoring the snapd state
+    if snap list | grep -E "broken$"; then
+        echo "snap in broken state"
+        exit 1
```

Comment:
Do we have enough context if this happens? Should we e.g. list all snaps?

Reply:
I think that's enough to see which snap is not being reset correctly.